tron/utils/scribereader.py

```diff
@@ -186,6 +199,14 @@ def read_log_stream_for_action_run(
     # XXX: for some reason, we're occasionally getting data out of order from scribereader - so we'll sort based on
     # timestamp until we can figure out what's causing this.
     output.sort(key=operator.itemgetter(0))
-    return [line for _, line in output] + (
-        [f"{malformed_lines} encountered while retrieving logs"] if malformed_lines else []
+    return (
+        [line for _, line in output]
+        + ([f"{malformed_lines} encountered while retrieving logs"] if malformed_lines else [])
+        + (
+            [
+                f"This output is truncated. Use this command to view all lines 'scribereader -s {paasta_cluster} -f stream_paasta_app_output_{namespace}_{job_name}__{action} --min-date {min_date} --max-date {max_date}'"
+            ]
+            if truncated_output
+            else []
+        )
     )
```

Comment:
this is probably long enough now that we wanna clean this up and do something like: 
```suggestion
    return (
        lines = [line for _, line in output]
        malformed = [f"{malformed_lines} encountered while retrieving logs"] if malformed_lines else []
        
        truncation_message = f"This output is truncated. Use this command to view all lines 'scribereader -s {paasta_cluster} -f stream_paasta_app_output_{namespace}_{job_name}__{action} --min-date {min_date} --max-date {max_date}'"
         truncated = [truncation_message] if truncated_output else []
         
         return lines + malformed + truncated
    )
```

rather than one-lining the whole thing